### ISSUE TITLE
Accuracy acheived on CAER-S dataset with provided glamor net weights

### ISSUE THREAD

**Issue by varsha-devi** [minhnhatvt/glamor-net#3]
Hello, thanks for the clear description for reproduction of the results. But when I used the pre-trained model to run evaluation on CAER-S test set, I acheived accuracy of 0.6527, which is not what is reported in table 2 of the paper 

 (i.e. GLAMOR-Net (original) 2021 --  77.90 ) (Image attached)
![Capture d’écran 2024-10-18 090130](https://github.com/user-attachments/assets/ec54b70f-b576-4188-9d01-40efd3a3c5b3)

link for the paper: https://link.springer.com/article/10.1007/s00521-021-06778-x

Can you please guide me what I have done wrong, as I do not have the same results on test dataset when using your pre-trained model.
